notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1593_force_actuator_error_elevation.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    for i in range(n_actuators):
</pre><p>The only detail to change here would be the parameter you are using to identify the force actuators. Right now, you are using the index. It would be better to have the actuator id. If you print the <code>FATable</code> variable, you will see something like:</p>

<pre class="ql-syntax" spellcheck="false">ForceActuatorData(index=42, actuator_id=143, x_position=1.60401001, y_position=-3.692780029, z_position=-2.158743, actuator_type=&lt;FAType.SAA: 0&gt;, subnet=&lt;FASubnet.C: 3&gt;, address=3, orientation=&lt;FAOrientation.NA: 0&gt;, x_index=None, y_index=None, z_index=42, s_index=None, near_neighbors=[140, 139, 134, 133, 138, 142], far_neighbors=[139, 140, 142, 138, 134, 133, 135, 141, 128, 137, 132, 129]), 
</pre><p>The <code>actuator_id</code> is what we should display.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Indeed! Thanks for catching this, I didn't know of this subtle detail. In fact, it led me to modifying the histogram before it as well.</p>